### ISSUE TITLE
frontend: assign a schema to a DI

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#53]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* open/select a schema and assign one or more DIs to it
* if #9 is implemented: for testing-purposes: issue test-mail to selected people (DIs) containing schema

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
